Week3/homework/step2-7.js

```diff
@@ -21,3 +25,7 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+
+//the first function just changes the integer but we console.log the x which is 9.
+//the second function takes the value 'y' which is an object. It takes the value of x but then it updates the x
+//to become 10 and indeed it updated the 'y'to become equal to 10 as well.
```

Comment:
I think you get it (?). Another example: 
```
let a = 4
let b = a
a += 1
What's the value of b?
```
vs objects...
```
let a = { name: 'gabe' }
b = a
a.country = 'us'
what's the value of b?
```
Do you see the difference?